docs/modding-basics/texture-replacement/index.md

```diff
@@ -0,0 +1,97 @@
+![Exported Files](assets/fmodel-5.png)
+
+Open DDS-Tools and do the following steps:
+1. Point DDS-Tools to the uasset file (`T_PhoneScreenWithAlpha_.uasset` in our case)
+2. Point DDS-Tools to the image file (`T_PhoneScreenWithAlpha_.png` in our case)
+3. Set an output folder, I recommend setting it to `ToolsDirectory/ModName/PAYDAY3/Content/Environment/_Common/Interactable/MobilePhone_01/Component` for reasons you will see in the packing stage. (The path may be different depending on the texture you are replacing). `ToolsDirectory` is where you placed the tools you downloaded in the prerequisites stage.
+4. Set UE version to `4.26 ~ 4.27`.
+
+
+
+After that you should have something like this:
+
+![DDS Tools](assets/ddstools.png)
+
+Click inject and you are ready for packing. The folder with the cooked texture should appear in the tools directory if you followed the recommendation.
+
+### Using Unreal Engine
+*This method is here mainly to show the simplest form of asset replacement that can be used for almost any type of asset.*
+
+Open Unreal Engine and create a new project, **name it PAYDAY3**.
+![Making a Project](assets/unreal-engine-1.webp)
+
+:::caution
+In Unreal Engine, paths already begin from `Content`, therefore you need to ignore PAYDAY3/Content when working inside Unreal Engine.
+:::
+
+Go to `Content Browser`, right click below and click on `New Folder` and replicate the directory structure of the asset you wish to replace.
+So the end result should be like:
+
+![Directory Structure](assets/unreal-engine-2.png)
+
+Now let's import our edited texture. The simplest way is to navigate to the directory and drag and drop your texture there.
+
+![Import Texture](assets/unreal-engine-3.png)
+
+Click `Save All` to ensure your files are saved.
+
+![Save All](assets/unreal-engine-4.png)
+
+Next step is to cook the files, simply click on `File` and then `Cook Content for Windows`
+
+![Cook Assets](assets/unreal-engine-5.png)
+
+Once done, the cooked files should be in `Documents\Unreal Projects\PAYDAY3\Saved\Cooked\WindowsNoEditor\PAYDAY3\Content` (Your Unreal Projects directory may be different).
+Copy the **folder** there into some folder with repak in it so that it's in `MyMod/PAYDAY3/Content/`.
+
+## Packing
+Packing is simple using Repak, however first you must make sure you replicate the pak path. In our case, the path is:
+`PAYDAY3/Content/Environment/_Common/Interactable/MobilePhone_01/Component` this is exactly how it should be before packing.
+
+If you followed through the guide, the directory structure should look like so:
+![Directory Structure](assets/directory-structure.png)
+
+I recommend placing repak.exe where the mod folder is:
+![Mod Folder](assets/repak-1.png)
+
+Hold shift and right click where repak.exe is, click on `Open PowerShell window here`. You will see a window popup.
+To pack your mod type the following command:
+```ps
+ ./repak pack ModName
+```
+![Repak Command](assets/repak-2.png)
```

Comment:
This image seems unnecessary. If you want to show the output, you can add it to the code block above.